HelpSource/Classes/Object.schelp

```diff
@@ -692,8 +692,129 @@ code::
 method::dump
 
 Print a detailed low level representation of the receiver to the post window.
+Except for the link::Classes/List:: class, this method is not overridden in other classes. Any of the MetaClasses, Classes and Instances can be used by this method.
+
+code::
+Meta_Object.dump // the meta class of the class Object
+::
+code::
+Object.dump      // the class called Object
+::
+code::
+Object.new.dump  // an istance of the class Object
+::
+
+discussion::
+
+list::
+## The detailed low level format and information varies depending on the receiver.
+## Some instance objects, especially unique objects, return the class name and value (also low data if necessary) of the dumped object:
+
+definitionlist::
+## link::Classes/Float::
+||
+code::
+1.0.dump
+::
+64-bit version of SuperCollider returns:
+teletype::
+   Float 1.000000   00000000 3FF00000
+-> 1.0
+::
+The last two groups of an 8-digit integer are the raw hexadecimal representation of the 64-bit double value according to link::https://ieeexplore.ieee.org/document/8766229##IEEE 754 Floating Point:: (https://ieeexplore.ieee.org/document/8766229). Each part is represented as follows:
```

Comment:
This information could go into `Float` help. Here it is perhaps too special and takes much space for that.

Reply:
Thanks, done!

I changed `Float` help document in this PR:
https://github.com/supercollider/supercollider/pull/6292/commits/cd5008216c2831587e2c50b645c6c34093ebaeef